apps/client/src/views/cuesheet/cuesheetCols.tsx

```diff
@@ -11,9 +11,25 @@ import { useCuesheetSettings } from './store/cuesheetSettingsStore';
 
 import style from './Cuesheet.module.scss';
 
-function makePublic(row: CellContext<OntimeRundownEntry, unknown>) {
-  const cellValue = row.getValue();
-  return cellValue ? <IoCheckmark className={style.check} /> : '';
+function MakePublic({ row, column, table }: CellContext<OntimeRundownEntry, unknown>) {
+  const update = useCallback(
+    (event: React.ChangeEvent<HTMLInputElement>) => {
+      // @ts-expect-error -- we inject this into react-table
+      table.options.meta?.handleUpdate(row.index, column.id, event.target.checked);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps -- we skip table.options.meta since the reference seems unstable
+    [column.id, row.index],
+  );
+
+  const event = row.original;
+  if (!isOntimeEvent(event)) {
+    return null;
+  }
+
+  const isChecked = event.isPublic;
+
+  //TODO: css styleing
+  return <Checkbox onChange={update} isChecked={isChecked} />;
```

Comment:
I think it looks ok, maybe just add
```ts
style={{ verticalAlign: 'middle' }}
```

Also, this field is a <Toggle> everywhere else, do we prefer a checkbox here?

Reply:
maybe, was trying to keep it looking as close to the old cuesheet